gpio/AM2302_sensor/AM2302.cpp

```diff
@@ -8,7 +8,7 @@
 #include "pico/stdlib.h"
 #include "AM2302-Sensor.hpp"
 
-const uint LED = 13U;
+const uint LED_PIN = 13U;
```

Comment:
Maybe you'd like to make use of the `PICO_DEFAULT_LED_PIN` define, so that your example doesn't require an external LED? https://github.com/raspberrypi/pico-sdk/blob/master/src/boards/include/boards/pico.h#L33
(see also e.g. https://github.com/raspberrypi/pico-examples/blob/master/blink/blink.c )